pixeltable/tests/test_dataframe.py

```diff
@@ -15,6 +15,14 @@ def test_select_where(self, test_tbl: catalog.Table) -> None:
         res2 = t.where(t.c2 < 10).select(t.c1, t.c2, t.c3).show(0)
         assert res1 == res2
 
+        res3 = t.where(t.c2 < 10).select(c1=t.c1, c2=t.c2, c3=t.c3).show(0)
```

Comment:
Should also have a test that the result doesn't change if you change the aliases.